lib/p2p/packets/types/SwapRequestPacket.ts

```diff
@@ -4,10 +4,6 @@ import PacketType from '../PacketType';
 export type SwapRequestPacketBody = {
   proposedQuantity: number;
   pairId: string;
-  takerAmount: number;
-  takerCurrency: string;
-  makerAmount: number;
-  makerCurrency: string;
```

Comment:
Removing these fields will also remove them from log (since we print the packet).

This will make debugging much harder. 

If you like to take it out, please include in the log something to replace it. 
Both on the maker and on the taker.

Reply:
Good point, I will add log statements every time they are calculated.